test/interval_tests/trig_tests.jl

```diff
@@ -75,7 +74,7 @@ facts("Trig tests") do
 
     @fact atan2(∅, entireinterval()) --> ∅
     @fact atan2(entireinterval(), ∅) --> ∅
-    @fact atan2(@interval(0.0, 1.0), @biginterval(0.0)) --> @biginterval(pi/2)
+    @fact atan2(@interval(0.0, 1.0), @biginterval(0.0)) --> @interval(pi/2)
```

Comment:
This is because of a fallback to a Julia definition `atan2(x, y)` when `x` and `y` have different types, which passes `x` and `y` through `float`. I think this only worked before because the `BigFloat` precision was set to 53; the type was previously `Interval{Float64}` as well, but we didn't realise!

If you want to insist on `@biginterval`s, presumably you want it everywhere, in which case this does the correct thing:

```
julia> atan2(@biginterval(0.0, 1.0), @biginterval(0.0))
[1.570796326794896619231321691639751442098584699687552910487472296153908203143099, 1.570796326794896619231321691639751442098584699687552910487472296153908203143116]₂₅₆
```